src/main/src/main/java/org/geoserver/catalog/CoverageViewResourceInfo.java

```diff
@@ -0,0 +1,13 @@
+/* (c) 2024 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.catalog;
+
+import org.geotools.data.DefaultResourceInfo;
+
+/** A ResourceInfo wrapper for CoverageViews */
+public class CoverageViewResourceInfo extends DefaultResourceInfo {
+
+    public CoverageViewResourceInfo(CoverageView coverageView) {}
```

Comment:
No need for this subclass, just return DefaultResourceInfo, it can be instantiated

Reply:
Done